src/components/container/ReasonsSection/ReasonsSection.scss

```diff
@@ -0,0 +1,81 @@
+.ReasonsSection {
+  width: 100%;
+  padding: 96px 126px;
+
+  &__contentWrapper {
+    max-width: 1260px;
+    width: 100%;
+    margin: auto;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 64px;
+  }
+
+  &__heading {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    padding: 0px;
+  }
+
+  &__heading h2 {
+    color: var(--color-bunting);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    &::after {
+      content: " ";
+      background-color: var(--color-blue-ribbon);
+      width: 24px;
+      height: 8px;
+      border-radius: 4px;
+      display: block;
+      margin-top: 5px;
+    }
+  }
+
+  &__cardsWrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0px;
+    gap: 96px;
+
+    &__firstRow,
+    &__secondRow {
```

Comment:
I think we need a margin may be? because there is no space in mobile view between 1st row and 2nd row

Reply:
I agree with Amindu here

Reply:
I have defined a gap there, but I'm not sure why it is not working

Reply:
The gap is only for the elements inside the rows. `.ReasonSection__cardsWrapper` does not have a gap atm

Reply:
ah, i have the culprit: A typo in the classname inside the .js file :)

Reply:
Ahh you're right! Easy to overlook that single missing letter :D